### DISCUSSION TITLE
UI Issue

### DISCUSSION THREAD

**Discussion by charlie1404** [community/community#63983] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

<img width="1774" alt="Screenshot 2023-08-17 at 3 10 45 AM" src="https://github.com/community/community/assets/20264337/dddeb39f-7b74-4820-86f8-97392102f40d">

username: charlie1404

Already tried hard refresh and reload with empty cache.
seems UI bug with new navigation

**Comment by lucasprag** [comment]
I see the same problem on multiple repos, but not all.
Examples:
- https://github.com/rubocop/ruby-style-guide
- https://github.com/standardrb/standard

![broken 2](https://github.com/community/community/assets/2293178/96720198-4e0a-4dee-9cd1-375d6216c450)
![broken](https://github.com/community/community/assets/2293178/e3296c29-3dfc-4abb-98b4-79f48efdb066)

**Comment by alexis-opolka** [comment]
@charlie1404 I had the same issue but it seems to be back to normal, for now.

Edit: There are indeed some repositories who are still bugged, I don't know why though. 🤔
